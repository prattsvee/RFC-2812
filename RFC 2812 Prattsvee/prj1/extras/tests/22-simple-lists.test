
[ :a, 22, :b ][]

[
  :some_atom12,
  # 99
  12
]
