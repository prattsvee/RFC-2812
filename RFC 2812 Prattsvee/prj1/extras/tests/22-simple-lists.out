[
  {
    "%k": "list",
    "%v": [
      {
        "%k": "atom",
        "%v": ":a"
      },
      {
        "%k": "int",
        "%v": 22
      },
      {
        "%k": "atom",
        "%v": ":b"
      }
    ]
  },
  {
    "%k": "list",
    "%v": []
  },
  {
    "%k": "list",
    "%v": [
      {
        "%k": "atom",
        "%v": ":some_atom12"
      },
      {
        "%k": "int",
        "%v": 12
      }
    ]
  }
]
